homeassistant/components/ituran/strings.json

```diff
@@ -0,0 +1,45 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "id_or_passport": "ID or passport number",
+          "phone_number": "Mobile phone number",
+          "mobile_id": "Mobile ID (leave empty if unknown)"
+        },
+        "data_description": {
+          "id_or_passport": "The goverment ID or passport number provided when registering with Ituran.",
+          "phone_number": "The mobile phone number provided when registering with Ituran. A one-time password will be sent to this mobile number.",
+          "mobile_id": "A unique ID used to identify the Ituran client. This can be shared between multiple clients to allow them all access to the Ituran services simultaniously. If one is not provided, it will be generated automatically."
+        }
+      },
+      "otp": {
+        "data": {
+          "otp": "OTP"
+        },
+        "data_description": {
+          "otp": "A one-time-password sent as a text message to the mobile phone number provided before."
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "invalid_otp": "OTP invalid",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
+      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]",
+      "id_mismatch": "ID or passport number are not allowed to change"
```

Comment:
If we don't want this to change, we should not ask the user for it.

If this requires a refactor of the reauth flow I suggest to keep the reauth for a followup PR

Reply:
Will handle this according to your response on the comment above

Reply:
Removed